kafkatest/README.md

```diff
@@ -0,0 +1,33 @@
+Contains kafkatest compatible clients for plugging in with the official Apache Kafka client tests
+
+
+Instructions
+============
+
+**Build both clients with statically linked librdkafka:**
+
+    $ cd go_verifiable_consumer
+    $ go build -tags static
+
+    $ cd go_verifiable_consumer
+    $ go build -tags static
+
+
+**Install librdkafka's dependencies on kafkatest VMs:**
+
+    $ cd ~/src/kafka  # your Kafka git checkout
+    $ for n in $(vagrant status | grep running | awk '{print $1}') ; do \
+      vagrant ssh $n -c 'sudo apt-get install -y libssl1.0.0 libsasl2-modules-gssapi-mit liblz4-1 zlib1g' ; done
```

Comment:
Not really important for now, but is the `static_all` option feasible on the original build machine? Seems way better, if it just involves installing a few packages, than having to install these things on all the test VMs.